src/packet_analysis/protocol/linux_sll/LinuxSLL.cc

```diff
@@ -0,0 +1,32 @@
+// See the file "COPYING" in the main distribution directory for copyright.
+
+#include "LinuxSLL.h"
+
+using namespace zeek::packet_analysis::LinuxSLL;
+
+LinuxSLLAnalyzer::LinuxSLLAnalyzer()
+	: zeek::packet_analysis::Analyzer("LinuxSLL")
+	{
+	}
+
+bool LinuxSLLAnalyzer::AnalyzePacket(size_t len, const uint8_t* data, Packet* packet)
+	{
+	auto len_sll_hdr = sizeof(SLLHeader);
+	if ( len_sll_hdr >= len )
+		{
+		packet->Weird("truncated_Linux_SLL_header");
+		return false;
+		}
+
+	//TODO: Handle different ARPHRD_types
```

Comment:
TODO. Is this something to do now or can it be extended after the merge?

Reply:
This is just a hint. Zeek never implemented all options here.